packages/client/src/Encrypt.ts

```diff
@@ -47,14 +47,19 @@ export default class PublisherEncryption implements Stoppable {
         }
 
         const streamId = streamMessage.getStreamId()
+
+        const isPublic = await this.streamEndpoints.isPublic(streamId)
+        if (isPublic) {
```

Comment:
Stop signal after `await`
```suggestion
        if (isPublic || this.isStopped) {
```